api/src/main/java/io/strimzi/api/kafka/model/KafkaListenerAuthentication.java

```diff
@@ -25,6 +26,11 @@
         @JsonSubTypes.Type(name = KafkaListenerAuthenticationScramSha512.SCRAM_SHA_512, value = KafkaListenerAuthenticationScramSha512.class),
 })
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Buildable(
```

Comment:
@tombentley When I compiled the `api` project I received an error that it can't find `KafkaListenerAuthenticationBuilder` in the generated sources of several types.  When I added this attribute the build error went away, but since this isn't an issue on master I'm confused as to why I had this problem.  TBH I didn't investigate this much more and I'm ignorant to what this attribute and the type it generates actually does.  If you do a `clean` compile on API do you have any trouble?

Reply:
Here is the error output:

```
100%: Builder generation complete.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/seglo/source/strimzi-kafka-operator/api/target/generated-sources/annotations/io/strimzi/api/kafka/model/KafkaListenerTlsFluentImpl.java:[11,13] cannot find symbol
  symbol:   class KafkaListenerAuthenticationBuilder
  location: class io.strimzi.api.kafka.model.KafkaListenerTlsFluentImpl<A>
[ERROR] /home/seglo/source/strimzi-kafka-operator/api/target/generated-sources/annotations/io/strimzi/api/kafka/model/KafkaListenerPlainFluentImpl.java:[11,13] cannot find symbol
  symbol:   class KafkaListenerAuthenticationBuilder
  location: class io.strimzi.api.kafka.model.KafkaListenerPlainFluentImpl<A>
[ERROR] /home/seglo/source/strimzi-kafka-operator/api/target/generated-sources/annotations/io/strimzi/api/kafka/model/KafkaListenerExternalLoadBalancerFluentImpl.java:[11,13] cannot find symbol
  symbol:   class KafkaListenerAuthenticationBuilder
  location: class io.strimzi.api.kafka.model.KafkaListenerExternalLoadBalancerFluentImpl<A>
[ERROR] /home/seglo/source/strimzi-kafka-operator/api/target/generated-sources/annotations/io/strimzi/api/kafka/model/KafkaListenerExternalNodePortFluentImpl.java:[11,13] cannot find symbol
  symbol:   class KafkaListenerAuthenticationBuilder
  location: class io.strimzi.api.kafka.model.KafkaListenerExternalNodePortFluentImpl<A>
[ERROR] /home/seglo/source/strimzi-kafka-operator/api/target/generated-sources/annotations/io/strimzi/api/kafka/model/KafkaListenerExternalRouteFluentImpl.java:[11,13] cannot find symbol
  symbol:   class KafkaListenerAuthenticationBuilder
  location: class io.strimzi.api.kafka.model.KafkaListenerExternalRouteFluentImpl<A>
```